modules/ignite-cli-2/src/main/java/org/apache/ignite/cli/core/exception/handler/EndOfFileExceptionHandler.java

```diff
@@ -0,0 +1,32 @@
+package org.apache.ignite.cli.core.exception.handler;
+
+import java.util.function.Consumer;
+import org.apache.ignite.cli.core.exception.ExceptionHandler;
+import org.apache.ignite.cli.core.exception.ExceptionWriter;
+import org.jline.reader.EndOfFileException;
+
+/**
+ * Exception handler for {@link EndOfFileException}.
```

Comment:
Can you justify what exactly this handler does? This is not clear why it does what it does.